### PR TITLE
Add password recovery functions to client

### PR DESCRIPTION
**Pull request by pradel**
#40

### CODE REVIEW

File: packages/client/src/AccountsClient.js [hunk 1]
```diff
@@ -269,6 +301,18 @@ const Accounts = {
   refreshSession(): Promise<void> {
     return this.instance.refreshSession();
   },
+  verifyEmail(token: string): Promise<void> {
+    return this.instance.verifyEmail(token);
+  },
+  resetPassword(token: string, newPassword: string): Promise<void> {
+    return this.instance.resetPassword(token, newPassword);
+  },
+  requestPasswordReset(userId: string, email?: string): Promise<void> {
```
Comment (TimMikeladze) [inline packages/client/src/AccountsClient.js]: Implementation question: Is it necessary to have `userId` passed in here? This code is being called on the client who isn't signed in, where does the userId come from?

**Comment by TimMikeladze** [comment]
Is this ready to merge?

**Comment by pradel** [comment]
I will add some tests before

**Comment by pradel** [comment]
Should be good now

**Pull request merged**